server.js

```diff
@@ -1,46 +1,46 @@
-var express = require('express');
-var fs = require('fs');
-var request = require('request');
-var cheerio = require('cheerio');
-var app = express();
+const express = require('express');
+const fs = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+const app = express();
+
+const URL = 'http://www.imdb.com/title/tt5052448/?ref_=fn_al_tt_1'
+const TITLE = '.title_wrapper > h1[itemprop="name"]'
+const RELEASE_DATE = '.title_wrapper > h1[itemprop="name"] a'
 
 
 app.get('/scrape', function(req, res) {
 
-  let url = 'http://www.imdb.com/title/tt5052448/?ref_=fn_al_tt_1'
+  request( URL, function(error, response, html) {
+    if (!error) {
+      const $ = cheerio.load(html);
+      const json = {}
+
+      $( TITLE ).filter(function() {
+        const title = $(this).text().trim()
 
-  request(url, function(error, response, html) {
-    if (!error){
-      var $ = cherrio.load(html);
-      var title, releaseDate, rating, moviePoster;
-      var json = {title: "", releaseDate: "", rating: "", moviePoster: ""};
+        json.title = title.substring( 0, title.indexOf( '(' ) - 1 )
+      })
 
-      $('.title_wrapper').filter(function() {
-        var data = $(this);
-        title = data.children().first().text();
-        releaseDate = data.children().last().text();
-        json.title = title;
-        json.releaseDate = releaseDate;
+      $( RELEASE_DATE ).filter(function() {
+        json.releaseDate = $(this).text();
       })
 
-      $('.subtext').filter(function() {
-        var data = $(this);
-        rating = data.children().first().text();
-        json.rating = rating;
+      $('.subtext > meta').filter(function() {
```

Comment:
Nice work traversing the dom and finding the children! See above about use of `filter()`